project.clj

```diff
@@ -1,19 +1,20 @@
-(defproject nomnom/duckula.monitoring "0.6.0"
+(defproject nomnom/duckula.monitoring "0.6.1-SNAPSHOT-1"
   :description "Monitoring Component for Duckula"
   :url "https://github.com/nomnom-insights/nomnom.duckula.monitoring"
   :license {:name "MIT License"
             :url "https://opensource.org/licenses/MIT"
             :year 2018
             :key "mit"}
-  :deploy-repositories {"clojars" {:sign-releases false
-                                   :username [:gpg :env/clojars_username]
-                                   :password [:gpg :env/clojars_password]}}
-  :dependencies [[org.clojure/clojure "1.10.1"]
-                 [com.stuartsierra/component "0.4.0"]
-                 [org.clojure/tools.logging "0.5.0"]
+
+  :deploy-repositories [["releases"  {:sign-releases false :url "https://clojars.org"}]
```

Comment:
Is that a new format for repos? It's different than this one for example: https://github.com/nomnom-insights/nomnom.bunnicula/blob/master/project.clj (or the one from lines 8-9